### PR TITLE
Make Enzyme tests Typescript(ES6) compatible

### PR DESCRIPTION
**Pull request by mtho11**
** Describe the change **
Make the Enzyme tests align with Typescript(ES6) import statements instead of CommonJS (require) statements.

### CODE REVIEW

File: src/setupTests.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import * as Enzyme from 'enzyme';
-require('jest-localstorage-mock');
-const Adapter = require('enzyme-adapter-react-16');
-
-Enzyme.configure({ adapter: new Adapter() });
+import { configure } from 'enzyme';
+import * as ReactSixteenAdapter from 'enzyme-adapter-react-16';
+const adapter = ReactSixteenAdapter as any;
```
Comment (josejulio) [inline src/setupTests.ts]: Is this really required?

Comment (mtho11) [reply src/setupTests.ts]: I believe it is. I copied this code from the Enzyme setup.

Comment (mtho11) [reply src/setupTests.ts]: @josejulio removed the `as any` and everything still works fine.

Comment (josejulio) [reply src/setupTests.ts]: Travis is complaining about something :(

**Pull request closed without merging**